enterprise/cypher/acceptance-spec-suite/src/test/resources/blacklists/cost-compiled.txt

```diff
@@ -157,6 +157,9 @@ Accessing a map with null should return null
 //SkipLimitAcceptance.feature
 Combining LIMIT and aggregation
 Using a optional match after aggregation and before an aggregation
+Limit before sort should work except in compiled
+Limit before top should work except in compiled
+Limit before distinct should work except in compiled
```

Comment:
I think these are a bit weird names for acceptance tests. It should not be a requirement that a certain implementation should not work.

Reply:
You've got a point there... Will be boarding my plane soon but will fix the names when I arrive home.

Reply:
Totally missed your comment btw but Andres just pinged me about it